### PR TITLE
Add a test for a react view

### LINKED ISSUE #490: Add a test for a react view
Find the easiest react view to test and test it. Involves setting up the tooling for react testing. Had some luck making actual tests with https://github.com/airbnb/enzyme so going to use it.

### PR DESCRIPTION
**Pull request by brandocalrissian**
Gets [enzyme](https://github.com/airbnb/enzyme) set up and gives us code coverage for [mediaThumbnail](https://github.com/18F/federalist/blob/bk-490/assets/app/components/mediaThumbnail.js) that actually checks how the rendering is affected based on what the component is given.

Simplified the mediaThumbnail component to make it a pure functional component after getting the tests in shape.

This fixes #490.

### CODE REVIEW

File: test/client/app/components/mediaThumbnailTest.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { shallow } from "enzyme";
+import { expect } from "chai";
+import { spy } from "sinon";
+
+import MediaThumbnail from "../../../../assets/app/components/mediaThumbnail";
+
+describe("mediaThumbnail", () => {
```
Comment (el-mapache) [inline test/client/app/components/mediaThumbnailTest.js]: Can we have a `forEach` block here that runs through the whitelist of acceptable extensions? Then we can reuse one `it` block and cut down on code duplication.

**Comment by el-mapache** [comment]
Looks pretty simple and straightforward! I like it.

File: test/client/app/components/mediaThumbnailTest.js (hunk 1 above)
Comment (brandocalrissian) [inline test/client/app/components/mediaThumbnailTest.js]: Good idea! Refactoring tests is a good thing. Will do.

**Comment by el-mapache** [comment]
Looks good, merge away.

🚢 🇮🇹

**Comment by brandocalrissian** [comment]
@el-mapache -- I took the refactor suggestion and added more refactoring while you were reviewing...

**Comment by brandocalrissian** [comment]
I'm done now.

**Comment by el-mapache** [comment]
yep, I saw it, still looking good!

**Comment by brandocalrissian** [comment]
👍 . Merging...

**Pull request merged**